applications/kodadot_assethub_nft_indexer_statemine_statemint.md

```diff
@@ -0,0 +1,210 @@
+| 7g.   | Metadata CREATE for NFT                     | Creating an indexing handler to set metadata for NFT from the chain.                                          |
+| 7h.   | Metadata CLEAR Handler                      | Developing a handler to index clearance of metadata for the collection from the chain.                        |
+| 7i.   | Metadata CLEAR for NFT                       | Introducing a handler for indexing clearance of metadata for NFT from the chain.                              |
+| 7j.   | Metadata IPFS Integration Handler     | Design a handler to retrieve IPFS Metadata from the IPFS network.                                                 |
+| 7l.   | Metadata Schema Development                | Creating and designing a GraphQL schema entity representing metadata.                                     |
+| 8a.   | NFT Royalties Schema Integration      |Design and include royalty support within the GraphQL schema.                                                  |
+| 8b.   | NFT Royalties Addition Handler        | Implement a handler to add royalty into NFT.                                                 |
+| 8c.   | NFT Royalties PAYOUT Handler           |Creation of handler to index royalty payout events from the chain.                                             |
+| 9.      | Fungible Assets Schema Creation      |Design and formulation of a GraphQL schema entity representing fungible assets.                                |
+| 9.      | Fungible Assets Force Creation        |Handlers will be developed to add system tokens like KSM/DOT into fungible assets.                             |
+| 9.      | Fungible Assets CREATE Event          |An event handler for indexing the creation of a fungible event from a chain, such as (RMRK/USDT) will be developed.|
+| 9.      | Metadata Support for Fungible Assets | Implement a handler to add metadata to a fungible asset event from the chain.                          |
+| 9c.   | Fungible Asset Allowlist Setup        |Setting up allows list-based indexing of fungible assets.                                                       |
+| 10a.  | Data Views Development                |Construction of data views for efficient access to indexed data.                                               |
+| 10b.  | Implementing Metadata Caching Layer |Develop and retry IPFS metadata if un-indexed by 7j.                                                               |
+| 11.    | Transfer of Collection Ownership    |Incorporate functionality to transfer collection ownership.                                                    |
+| 12a.  | Allocation of Initial Storage Bandwidth |Provide an initial bandwidth budget for IPFS item fetching (Payment is based on bandwidth scale).               |
+| 13b.  | Initial Storage Budget Allocation     |Allocate initial storage budget of $5,000 (Equivalent to 100GB storage at $5 per GB) for initial items minting, allowing for future data expansion.|
```

Comment:
I'm afraid these [aren't covered by the grants program](https://grants.web3.foundation/docs/faq#what-activitiespositions-do-you-fund).

Reply:
Sure, we can subtract that. I think this was counted in the first version of this revision of this proposal.